Task1/1.2/formatter.js

```diff
@@ -0,0 +1,10 @@
+import { Transform } from 'stream';
+
+export default new Transform ({
+  transform: (data, encoding, callback) => {
+    const entries = Object.entries(JSON.parse(data.toString('utf8')))
+        .filter(([key]) => key != 'Amount')
+        .map(([key, value]) => [key.toLowerCase(), key === 'Price' ? parseFloat(value) : value]);
```

Comment:
For Price fields I would suggest to try to use `colParser` property from configuration object from `csvtojson` package. It is not required, just another way of implementation

Reply:
Also you can read/try to use csv().subscribe method. It also allows you to make some transformations, but a bit in another way

Reply:
ok, will know